app/util.spec.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const util = require('./util');
+const expect = require('chai').expect;
+
+describe('util module', () => {
```

Comment:
With more time, I'd add mocking of some of the more complex behavior like the HTTP requests from request/request-promise.